app/assets/stylesheets/application.css.scss

```diff
@@ -127,3 +89,9 @@ $image-path: '';
     display: block;
   }
 }
+
+.usa-button--tiny {
```

Comment:
this button variant was removed from the Login.gov design system, but we don't have an alternative for it currently, so i just ported the css